### PR TITLE
Add pyret

### PR DESCRIPTION
**Pull request by BNAndras**

### CODE REVIEW

File: tests/data/pyret.arr [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//! 22 lines 8 code 3 comments 6 blanks
```
Comment (glennj) [inline tests/data/pyret.arr]:
```suggestion
# 22 lines 8 code 3 comments 6 blanks
```
Would want a Pyret comment there.

Comment (BNAndras) [reply tests/data/pyret.arr]: Thanks, I’m not sure how the copy and paste mangled that.

**Comment by ErikSchierboom** [comment]
CI is failing

File: languages.json [hunk 2]
```diff
@@ -1148,6 +1148,13 @@
       "multi_line_comments": [["{-", "-}"]],
       "extensions": ["purs"]
     },
+    "Pyret": {
+      "line_comment": ["#"],
+      "multi_line_comments": ["#|", "|#"],
```
Comment (glennj) [inline languages.json]:
```suggestion
      "multi_line_comments": [["#|", "|#"]],
```
I see it now: this should be a list of lists.

**Comment by BNAndras** [comment]
Still failing so I’ll take a closer look at both PRs Friday.

File: tests/data/pyret.arr [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# 22 lines 8 code 3 comments 6 blanks
```
Comment (glennj) [inline tests/data/pyret.arr]:
```suggestion
# 22 lines 9 code 8 comments 5 blanks
```
I think tokei is counting _lines_ of comments: #lines = #code + #comments + #blanks

**Comment by ErikSchierboom** [comment]
Great!

**Pull request merged**